### DISCUSSION TITLE
How to use bat as previewer?

### DISCUSSION THREAD

**Discussion by augustocdias** [nvim-telescope/telescope.nvim#1435] [Q&A]
I see in the docs that the previewers that use bat are deprecated...

I have it configured like this currently:

```lua
require('telescope').setup({
    defaults = {
        file_previewer = require('telescope.previewers').cat.new,
        grep_previewer = require('telescope.previewers').vimgrep.new,
        qflist_previewer = require('telescope.previewers').qflist.new,
    },
})
```

How should I configure it then if I want to use bat instead of the default vim buffer?

The reason I prefer bat is because I'm getting too many timeouts and files without highlighting with the default method.

Another question is: how do I disable messages from it? I'm seeing things like this when I open/close telescope and it is distracting me:

```
1 line less; before #151  10 seconds ago
"ledger/src/lib.rs" 262 lines --54%--
"[Prompt]" [+] 1 line --100%--
```

I'd be fine to see messages such as `No entries for telescope`, but those that are showing is useless for me.

**Comment by Conni2461** [comment]
deprecated in terms of we dont maintain them anymore. Because otherwise its too much. They still work (at least for me, i just tried).

> how do I disable messages from it

I didnt see any messages. They look like less messages. Can you share more things. Screeshot, config, `$LESS` environment var

**Reply by augustocdias** [reply]
They're working for me... But by deprecated I imagine it will be removed at some point... And the buffer solution is being way slower and sometimes I'm not getting highlighting even for filetypes supported by treesitter. 

And those messages are not from telescope indeed. I'm seeing them every time a buffer is hidden...
